src/shared/lib/getCanonicalMutationType.ts

```diff
@@ -98,6 +98,7 @@ export function getProteinImpactTypeFromCanonical(mutationType:CanonicalMutation
         case "fusion":
         case "silent":
```

Comment:
I think `default` is a good idea, I'm just explaining why we should still keep the other cases (fusion, silent..) even though logically it would be equivalent to remove them